### PR TITLE
Version

### PR DESCRIPTION
**Pull request by mslacken**
Added subcommand `wwctl version`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .PHONY: all
-
 VERSION := 4.2.0
+VERSION_FULL ?= $(shell test -e .git && git describe --tags --long)
```
Comment (PaulCharlton) [inline Makefile:3]: @mslacken is it possible that "git describe" should only have the "--first-parent" option set?  see: https://git-scm.com/docs/git-describe/2.33.0

**Comment by mslacken** [comment]
Right `--first-parent` is sensible, I have updated the PR

**Comment by gmkurtzer** [comment]
merge conflicts... let's blame @jcsiadal  instead of me. ;-)

**Comment by mslacken** [comment]
Rebased it now, so that it can be merged

**Pull request merged**